invenio_accounts/models.py

```diff
@@ -82,6 +103,16 @@ class User(db.Model, Timestamp, UserMixin):
     version_id = db.Column(db.Integer, nullable=False)
     """Used by SQLAlchemy for optimistic concurrency control."""
 
+    _user_profile = db.Column(
+        "user_profile", json_field, default=lambda: dict(), nullable=True,
```

Comment:
Perhaps change name to from ``user_profile`` to ``profile`` since it's already on the user class or is it going to conflict with an existing property?
```suggestion
        "profile", json_field, default=lambda: dict(), nullable=True,
```

Reply:
true. there's no reason that i see right now why the DB column couldn't be called `profile` already; it's just the property name on the model class that'd collide with the `profile` relationship to the current `invenio-userprofile` model